src/main/webapp/app/modeling-assessment-editor/modeling-assessment-editor.component.html

```diff
@@ -37,16 +37,18 @@ <h3 class="top-container">
             <button class="btn ml-2 btn-success" [hidden]="!conflicts" (click)="onShowConflictResolution()">
                 <span jhiTranslate="modelingAssessmentEditor.button.resolveConflict">Resolve Conflict</span>
             </button>
-            <button class="btn ml-2 btn-success" *ngIf="result?.completionDate && isAuthorized" [disabled]="busy" (click)="assessNextOptimal()">
+            <button class="btn ml-2 btn-success" *ngIf="result?.completionDate && isAssessor" [disabled]="busy" (click)="assessNextOptimal()">
                 <fa-icon *ngIf="busy" icon="spinner" [spin]="true">&nbsp;</fa-icon>
                 <span jhiTranslate="modelingAssessmentEditor.button.nextSubmission"> Next Submission</span>
```

Comment:
The `Assess Next Submission` button is visible when resolving a more feedback request. Please hide the button in this case as well. You already did this for text submissions.

Reply:
Good suggestion. Interesting that this thing was there before and nobody noticed